zebra-state/src/service/non_finalized_state/tests/prop.rs

```diff
@@ -451,6 +451,7 @@ fn finalized_equals_pushed_history_tree() -> Result<()> {
 /// Check that rejected blocks do not change the internal state of a genesis chain
 /// in a non-finalized state.
 #[test]
+#[cfg(not(target_os = "windows"))]
```

Comment:
Did this one fail? It's passing for me locally.

Reply:
It seems i got confused, this test is passing the widows CI. I restored it at https://github.com/ZcashFoundation/zebra/pull/8393/commits/60b43c0ff4a8c4c8305141c18ac3709712f75886